src/Module.Client/GUI/Scoreboard/CrpgMissionScoreboardVM.cs

```diff
@@ -106,6 +112,45 @@ public CrpgMissionScoreboardVM(bool isSingleTeam, Mission mission)
             _missionPollComponent = mission.GetMissionBehavior<MultiplayerPollComponent>();
         }
 
+        _commanderClient = mission.GetMissionBehavior<CrpgCommanderBehaviorClient>();
+        if (_commanderClient != null)
+        {
+            _commanderPollComponent = mission.GetMissionBehavior<CrpgCommanderPollComponent>();
+
+            if (_commanderPollComponent != null)
+            {
+                _canStartCommanderPolls = true;
+            }
+            else
+            {
+                _canStartCommanderPolls = false;
+            }
+        }
+        else
+        {
+            _canStartCommanderPolls = false;
+        }
```

Comment:
This can be refactored to 
```
_commanderClient = mission.GetMissionBehavior<CrpgCommanderBehaviorClient>();
_canStartCommanderPolls = false;
if (_commanderClient != null && _commanderPollComponent != null)
        {
        _canStartCommanderPolls = true;
        }
```

Reply:
Done, I added 
`_commanderPollComponent = mission.GetMissionBehavior<CrpgCommanderPollComponent>();`